### PR TITLE
5 server model

### PR DESCRIPTION
**Pull request by ghost**
Created and Updated Server Model

### CODE REVIEW

File: moninag/server/models.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    """
+    name = models.CharField(max_length=50)
+    address = models.CharField(max_length=200)
+    status = models.CharField(max_length=100, choices=STATUS_CHOICES, default=0)
+
+    def __str__(self):
+        return "ServerId: {}, ServerName: {}, ServerAddress: {}, ServerStatus {}".format(self.id,
+                                                                                         self.name,
+                                                                                         self.address,
+                                                                                         self.status)
```
Comment (lhalam) [inline moninag/server/models.py:27]: add end line

File: moninag/server/models.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    (1, ("NotSelected")),
+    (2, ("Production")),
+    (3, ("Staging")),
+)
+
+
+class Server(models.Model):
+    """
+    Server model with fields:
+    id, name, address, status
```
Comment (lhalam) [inline moninag/server/models.py]: write more detailed docstring

File: moninag/server/models.py (hunk 1 above)
Comment (ghost) [reply moninag/server/models.py:27]: added

**Review by byshovets — CHANGES_REQUESTED** [review]
Squash all your commits together.

**Pull request closed without merging**